marcdomain/Fast-Food-Fast
# #160921211 Ft fetch specific user orders from admin dashboard

### What does this PR do?
- Implementation of fetch API on getting specific user order-history at admin dashboard
### Description of Task to be completed
- Use fetch API to consume the get-specific-user-order-history endpoint at the admin dashboard
### How should this be manually tested?
- Using a web browser, access https://marcdomain.github.io/Fast-Food-Fast/views/  to Login  as admin _**email**_ admin@gmail.com _**password**_ adminuser
- Navigate to https://marcdomain.github.io/Fast-Food-Fast/views/orders-admin.html  view all orders.
- Click on _**phone**_ in any order row to see all orders made by that specific user.
### Any background context you want to provide?
- Authentication is required to view order history
### What are the relevant pivotal tracker stories?
- [#160921211](https://www.pivotaltracker.com/story/show/160921211)
